<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MAME Plus! GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QSplitter" name="splitter_2" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QComboBox" name="comboBox" >
            <item>
             <property name="text" >
              <string>All Arcades</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Available Arcades</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>All Consoles</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_2" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSearch" >
          <property name="statusTip" >
           <string>type a keyword</string>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="treeViewGameList" />
        </item>
        <item>
         <widget class="QProgressBar" name="progressBarGamelist" />
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTabWidget" name="tabWidget" >
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_3" >
         <attribute name="title" >
          <string>Option</string>
         </attribute>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QTreeView" name="treeViewOption" >
            <property name="editTriggers" >
             <set>QAbstractItemView::CurrentChanged|QAbstractItemView::EditKeyPressed|QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors" >
             <bool>true</bool>
            </property>
            <property name="rootIsDecorated" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabSnapshot" >
         <attribute name="title" >
          <string>Snapshot</string>
         </attribute>
         <layout class="QGridLayout" />
        </widget>
        <widget class="QWidget" name="tabFlyer" >
         <attribute name="title" >
          <string>Flyer</string>
         </attribute>
         <layout class="QGridLayout" />
        </widget>
        <widget class="QWidget" name="tabCabinet" >
         <attribute name="title" >
          <string>Cabinet</string>
         </attribute>
        </widget>
       </widget>
       <widget class="QTabWidget" name="tab_x" >
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_2" >
         <attribute name="title" >
          <string>FrontendLog</string>
         </attribute>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QTextBrowser" name="textBrowserFrontendLog" />
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab" >
         <attribute name="title" >
          <string>MAMELog</string>
         </attribute>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QTextBrowser" name="textBrowserMAMELog" />
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabHistory" >
         <attribute name="title" >
          <string>History</string>
         </attribute>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QTextBrowser" name="textBrowser_2" />
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabMAMEInfo" >
         <attribute name="title" >
          <string>MAMEInfo</string>
         </attribute>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QTextBrowser" name="textBrowser_3" />
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabStory" >
         <attribute name="title" >
          <string>Story</string>
         </attribute>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionExitStop" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>View</string>
    </property>
    <addaction name="actionRefresh" />
   </widget>
   <widget class="QMenu" name="menuOptions" >
    <property name="title" >
     <string>Options</string>
    </property>
    <addaction name="actionDefault_Options" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuView" />
   <addaction name="menuOptions" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="movable" >
    <bool>false</bool>
   </property>
   <property name="iconSize" >
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh" />
   <addaction name="actionDefault_Options" />
   <addaction name="actionAbout" />
  </widget>
  <action name="actionDefault_Options" >
   <property name="icon" >
    <iconset resource="qmc2main.qrc" >:/res/32x32/applications-system.png</iconset>
   </property>
   <property name="text" >
    <string>Default Options</string>
   </property>
  </action>
  <action name="actionRefresh" >
   <property name="icon" >
    <iconset resource="qmc2main.qrc" >:/res/32x32/view-refresh.png</iconset>
   </property>
   <property name="text" >
    <string>Refresh</string>
   </property>
   <property name="shortcut" >
    <string>F5</string>
   </property>
  </action>
  <action name="actionExitStop" >
   <property name="text" >
    <string>Exit</string>
   </property>
   <property name="shortcut" >
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="icon" >
    <iconset resource="qmc2main.qrc" >:/res/32x32/help-browser.png</iconset>
   </property>
   <property name="text" >
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qmc2main.qrc" />
 </resources>
 <connections/>
</ui>
